packages/tenderly-core/src/internal/core/services/TenderlyService.ts

```diff
@@ -112,19 +135,22 @@ export class TenderlyService {
     username: string
   ): Promise<void> {
     if (!TenderlyApiService.isAuthenticated()) {
-      console.log(`Error in ${this.pluginName}: ${ACCESS_TOKEN_NOT_PROVIDED_ERR_MSG}`);
+      logger.error(`Error in ${this.pluginName}: ${ACCESS_TOKEN_NOT_PROVIDED_ERR_MSG}`);
       return;
     }
 
     const tenderlyApi = TenderlyApiService.configureInstance();
     try {
+      logger.debug("Making a call to the api...");
       const res = await tenderlyApi.post(`/api/v1/account/${username}/project/${tenderlyProject}/contracts`, {
         ...request,
       });
+      logger.debug("Call to the api successfully made.");
```

Comment:
Same this is not useful information.
Some formats in that situation that we want to look at can be like:
1. Trying to get project contracts
2. if Yes - Successfully got project contracts
3. if No - Failed to get project contracts

Of course not exactly how I write it it need a little bit more polishing but some format like that.